conans/client/remote_manager.py

```diff
@@ -220,9 +219,9 @@ def _call_remote(self, remote, method, *args, **kwargs):
 def uncompress_file(src_path, dest_folder, scope=None):
     try:
         filesize = os.path.getsize(src_path)
-        big_file = filesize > 1000000  # 10 Mb
+        big_file = filesize > 10000000  # 10 Mb
```

Comment:
Oops nice catch! This is why I prefer doing `10 * 1 << 20` for things like this.

Also, got confused with the comment:
```suggestion
        big_file = filesize > 10000000  # 10 MB
```